librz/core/cbin.c

```diff
@@ -3236,6 +3236,9 @@ RZ_API bool rz_core_bin_info_print(RZ_NONNULL RzCore *core, RZ_NONNULL RzBinFile
 			free(buf);
 			pj_end(pj);
 		}
+		if (info->linker) {
+			pj_ks(pj, "lnkrver", info->linker);
```

Comment:
i prefer a full name `linkerver` @XVilka @tushar3q34  what do you think?

Reply:
Yes we can do that, we would just have to put extra spaces in all the tests printing tables. Should I do that ?

Reply:
that is normal. yes, please fix the tests.

Reply:
@wargio So should I start working on changing the name to `linkerver` ?

Reply:
Ok, i gave a thought and probably we should just use the `compiler` field.

change that `rz_str_newf("%02d.%02d", major, minor);` into `rz_str_newf("Linker %02d.%02d", major, minor);`

Reply:
It runs perfectly for all except one test of `golang`
`[XX] db/analysis/golang Resolve all symbols on a stripped PE go1.12 binary (x64)`
```
' bins/golang/hello_go_strip.exe
-- stdout
--- expected
+++ actual
@@ -1,7 +1,7 @@
 1788
 0x004a699b str.hello__hacktivity
 9222
-compiler go1.15
+compiler Linker 03.00
 errors
 fmt
 go
```
Here the compiler is supposed to contain `go1.15` but it is instead containing `Linker 03.00`.

Reply:
do this, if the field is already populated, then append the string.

Reply:
this probably should be done in the golang code.